op-node/rollup/engine/engine_controller.go

```diff
@@ -45,26 +46,23 @@ type ExecEngine interface {
 	L2BlockRefByHash(ctx context.Context, hash common.Hash) (eth.L2BlockRef, error)
 }
 
-type ECMetrics interface {
-	derive.Metrics
-	RecordL2Ref(name string, ref eth.L2BlockRef)
-}
-
 type SyncDeriver interface {
 	OnELSyncStarted()
 }
 
 type EngineController struct {
 	engine     ExecEngine // Underlying execution engine RPC
 	log        log.Logger
-	metrics    ECMetrics
+	metrics    opmetrics.Metricer
```

Comment:
What do you think about adding the necessary methods to `ECMetrics` to avoid making `EngineController` dependent on an unnecessarily large interface?

Reply:
Keeping metrics as is for now to keep PR size small